scripts/install_zds.sh

```diff
@@ -262,6 +262,18 @@ if  ! $(_in "-node" $@) && ( $(_in "+node" $@) || $(_in "+base" $@) || $(_in "+f
             echo $ACTIVATE_NVM >> $ZDS_ENV/bin/activate
             echo $ACTIVATE_NVM >> $ZDS_ENV/bin/activate.csh
             echo $ACTIVATE_NVM >> $ZDS_ENV/bin/activate.fish
+
+            # bash compatibility for nvm with fish
+            # see https://eshlox.net/2019/01/27/how-to-use-nvm-with-fish-shell
+            if [[ $SHELL =~ "fish" ]]; then
+                curl https://git.io/fisher --create-dirs -sLo ~/.config/fish/functions/fisher.fish
```

Comment:
A quoi sert ce fichier ? Est-il indispensable ?

Reply:
Il sert à installer Fisher et oui dans l'état, il est indispensable. Je te laisse te référer au lien que j'ai mis dans la PR : [https://eshlox.net/2019/01/27/how-to-use-nvm-with-fish-shell](https://eshlox.net/2019/01/27/how-to-use-nvm-with-fish-shell) pour plus de détail.

Reply:
Est ce vraiment utile d'installer fisher ? Ce n'est pas à l'utilisateur de le faire ?

Reply:
Je veux dire si je suis un utilisateur de fisher, je l'ai déjà d'installé sur ma machine non ?

Reply:
C'est idempotent.